### PR TITLE
fix(DTFS2-6056): All facilities - fix dealId conversion to string

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue: 
* MIN facilities do not show up on all facilities tab
* MIN tfm-deal dealSnapshot._id converted to string instead of ObjectId

# Changes made:
* returned dealUpdate from `updatePortalDealFromMIAtoMIN` as compared to returning full deal 
* removed full update body from update in `updateDealSnapshot` 
* Added extra tests

### CODE REVIEW

File: trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js [hunk 1]
```diff
@@ -654,7 +654,7 @@ describe('/v1/deals', () => {
 
         expect(status).toEqual(200);
         expect(body.submissionType).toEqual(CONSTANTS.DEALS.SUBMISSION_TYPE.MIN);
-        expect(typeof body.manualInclusionNoticeSubmissionDate).toEqual('number');
+        expect(typeof body.manualInclusionNoticeSubmissionDate).toEqual('string');
```
Comment (abhi-markan) [inline trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js:657]: Why is this changed to `string` please?

File: trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.api-test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  it('should return the dealUpdate object on BSS deal update', async () => {
+    const update = await updatePortalDealFromMIAtoMIN(dealId, CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS, checker);
+
+    const dealUpdate = {
+      submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIN,
+      details: {
+        checkerMIN: checker,
+        manualInclusionNoticeSubmissionDate: expect.any(String),
```
Comment (abhi-markan) [inline trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.api-test.js:46]: LOC:33 and :46 should be `number`?

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Two minor comments to address please.

File: trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.api-test.js (hunk 2 above)
Comment (Zainzzkk) [reply trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.api-test.js:46]: it's using toString() on the now function

File: trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js (hunk 1 above)
Comment (Zainzzkk) [reply trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js:657]: now does .toString() so not sure why it was a number in the first place

**Review by abhi-markan — APPROVED** [review]
